frontend/app/query-client.tsx

```diff
@@ -3,13 +3,12 @@
 import React from "react";
 
 import { QueryClientProvider as InternalProvider, QueryCache, QueryClient } from "@tanstack/react-query";
-import { toast } from "sonner";
 
 const queryClient = new QueryClient({
   queryCache: new QueryCache({
     onError: (error, query) => {
       if (query.state.data !== undefined) {
-        toast.error(
+        console.log(
```

Comment:
Dann benutz bitte `console.error` :)

```suggestion
        console.error(
```

Reply:
Ah damn fair